src/test/adapter.ts

```diff
@@ -9,7 +9,7 @@ const assert = chai.assert;
 
 describe('PHP Debug Adapter', () => {
 
-    const TEST_PROJECT = path.normalize(__dirname + '/../../testproject');
+    const TEST_PROJECT = path.normalize(__dirname + '/../../testproject/main');
```

Comment:
I'm assuming that you're requesting a test that involves `/testproject/secondary`? Please give me a little direction on this.